contracts/OwnedRegistrar.sol

```diff
@@ -1,9 +1,19 @@
-pragma solidity ^0.4.20;
+pragma solidity >=0.5.0 <0.7.0;
 pragma experimental ABIEncoderV2;
 
 import "@ensdomains/ens/contracts/ENS.sol";
 import "./RBAC.sol";
-import "./OwnerResolver.sol";
+import "@ensdomains/resolver/contracts/Resolver.sol";
+
+interface OldENS {
```

Comment:
It's safe to use the `ENS` interface for `oldENS`; just don't call any of the newly added methods.

Reply:
Fixed.